js/controllers/hotkeys.js

```diff
@@ -37,77 +45,77 @@ function HotkeysController(windowController, tabs, editor, settings, analytics)
 HotkeysController.prototype.onKeydown_ = function(e) {
   if (e.ctrlKey || e.metaKey) {
     switch (e.keyCode) {
```

Comment:
Way easier than this! Let's stop using `e.keycode` and start using `e.key`!

https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key

also note that the [compatibility table](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key#Browser_compatibility) says that it should be fine for our versions of Chrome so I think this is a much better way to handle this :)

Reply:
Done, thanks for pointing that out!